frontend/components/global/InputQuantity.vue

```diff
@@ -10,8 +10,9 @@
       :min="min"
       :max="max"
       type="number"
-      size="small"
       variant="plain"
+      density="compact"
+      style="width: 60px;"
```

Comment:
`size="small"` throws a warning in the console that it's invalid (even though it works??) and setting the density and width here mimics the same behavior. Note that `width: 60px;` copies the `max-width` setting a few lines above this.